Cargo.toml

```diff
@@ -25,6 +25,7 @@ smol_socket = ["netlink-proto/smol_socket", "async-std"]
 [dependencies]
 anyhow = "1.0.44"
 async-std = { version = "1.9.0", optional = true}
+bilge = "0.2.0"
```

Comment:
We already have similar discussion on using bit flag crate in https://github.com/rust-netlink/netlink-packet-route/pull/62

To summery on that, I do not want to use any bit flag crate, but prefer `Vec<XxxFlag>` and hide the fact of bitflag
to user.

Reply:
I'm OK with not including external crates, but sooner or later we'll need to write an internal macro to automate the desired interface since there are so many bit fields in ieee80211 standard which are needed by the scan structs. Many are even multi-bit.

Reply:
I am OK to internal macros for auto generate bit fields

Reply:
Base on our latest decision in https://github.com/rust-netlink/netlink-packet-route/pull/62 , you could use `bitflags` for those bit fileds.